.travis.yml

```diff
@@ -4,8 +4,8 @@ matrix:
     - jdk: openjdk11
 
 script: >-
-    ./config/travis/run-checks.sh &&
-    ./gradlew clean checkstyleMain checkstyleTest test coverage coveralls asciidoctor
+  ./config/travis/run-checks.sh &&
```

Comment:
Travis is up and running agn, is this safe?

Reply:
Spacing issue, otherwise it' is fine.